Source/Python/adapter/request_handling/handlers.py

```diff
@@ -1,92 +1,116 @@
 from http import HTTPStatus
 from typing import Tuple, Dict, Union, List, Any
 
-from jsonpatch import JsonPatch, InvalidJsonPatch, PatchOperation, make_patch, JsonPatchTestFailed
-
-from adapter.persistence.repositories import InMemoryLocationRepository
-from adapter.request_handling.utils import error_response, parse_optional_tag_query_param
-from adapter.views import LocationView, LocationIdView
-from usecase.locations_usecases import LocationUseCase
+from adapter.persistence.in_memory_repositories import InMemoryLocationRepository, InMemoryTravelerRepository
```

Comment:
This shouldn't be accessing the `in_memory_repositories` module. The initialization of these should be moved up to the application construction level

Reply:
fixed